### PR TITLE
Add 7Zip icon

### LINKED ISSUE #5872: 7-Zip
**Brand Name:** 7-Zip
**Website:** https://www.7-zip.org
**Alexa rank:** 12,265
**Official resources for icon and color:** _Check this out_ - `#000000`. See below in the 7-Zip homepage.

### PR DESCRIPTION
**Pull request by zrfisaac**
7zip

**Issue:** Close #5872

**Similarweb rank:** [27,788](https://www.7-zip.org/download.html)

### Checklist
  - [X] I updated the JSON data in `_data/simple-icons.json`
  - [X] I optimized the icon with SVGO or SVGOMG
  - [X] The SVG `viewbox` is `0 0 24 24`

### Description
7-Zip is a file archiver with a high compression ratio.

### CODE REVIEW

**Comment by sachinraja** [comment]
Yes, we do need more info on how you obtained the SVG icon (for your other PRs too)

**Comment by zrfisaac** [comment]
Year I didn't find the icon either, there is no SVG official file, so I made one by myself.
Now I have a question, can I do that?
Or must exist an official publication of the SVG file?

**Comment by zrfisaac** [comment]
I've created An [issue](https://github.com/jinfeihan57/p7zip/issues/175) in the official 7zip github repository to upload the SVG file with the logo.
You can see in [7zip page](https://www.7-zip.org/links.html) that this [repository](https://github.com/jinfeihan57/p7zip) is official :
image
Or you need that this SVG file must be uploaded in [Source Forge](https://sourceforge.net/projects/sevenzip/) repository ?

**Comment by sachinraja** [comment]
I think we can use Source Forge as a source so that could work

**Comment by zrfisaac** [comment]
I asked them, but apparently they don't have a svg file.
I’m working with that.
After they publish one I will ask you to merge.

Wait a while until I get it done.
🙌

**Comment by antermin** [comment]
jinfeihan57's repo isn't really official - being mentioned in the official 7-Zip website does not make it official, and it is actually a fork of p7zip, while p7zip is a port of 7-Zip to POSIX systems.

The official repo should be the one [on SourceForge by Igor Pavlov](https://sourceforge.net/projects/sevenzip/).

-----

Igor Pavlov created 7-Zip but it was Windows-only, so someone ported 7-Zip to POSIX systems (Linux, macOS, etc.) as [p7zip](https://sourceforge.net/projects/p7zip/).

However, p7zip is abandoned for several years already (no updates since 2016), so jinfeihan57 started a new fork. A few Linux distros (e.g. Arch Linux) switched to jinfeihan57's fork.

As Igor Pavlov has recently added official Linux/BSD/macOS support in 7-Zip, openSUSE Tumbleweed dropped p7zip and switched to upstream 7-Zip 21.07, and Debian, FreeBSD, etc. are now offering 7-Zip 21.07.

**Comment by zrfisaac** [comment]
Okey, I'm working on updating the official repository with this svg file, I will inform you when it is done.

**Comment by adamrusted** [comment]
Hey @zrfisaac - any joy getting an official SVG hosted for 7Zip?
No stress if not - just running through older PRs!

**Comment by zrfisaac** [comment]
> Hey @zrfisaac - any joy getting an official SVG hosted for 7Zip? No stress if not - just running through older PRs!

I've been off these days due to my job, but I'm back now. I will work on this and answer you as soon as possible! 🙌

**Comment by PeterShaggyNoble** [comment]
Any update on this, @zrfisaac? If not then I think we should just proceed with reviewing this PR as-is.

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
Would you mind having another look at this, please, @zrfisaac? In the absence of anything better to go on, we'll have to run with [their primary logo](https://www.7-zip.org/7ziplogo.png) and I'm seeing [major differences](https://github.com/simple-icons/simple-icons/assets/15157491/2d565cf9-3923-4a9c-b7a3-815b99cf523d) in your version , in red.

There's actually [an SVG on Wikimedia](https://commons.wikimedia.org/wiki/File:7ziplogo.svg) that might provide a better starting point.

File: _data/simple-icons.json [hunk 1]
```diff
@@ -55,6 +55,11 @@
             "hex": "0099E5",
             "source": "https://about.500px.com/press"
         },
+        {
+            "title": "7Zip",
+            "hex": "000000",
+            "source": "https://www.7-zip.org/download.html"
```
Comment (PeterShaggyNoble) [inline _data/simple-icons.json:61]:
```suggestion
            "source": "https://commons.wikimedia.org/wiki/File:7ziplogo.svg"
```

File: icons/7zip.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>7Zip</title><path d="M0 4.479v15.043h24V7.836H10.757V4.479zm1.39 1.326h8.074V7.48c-.665.638-1.345 1.557-2.033 2.755a15.95 15.95 0 0 0-1.58 3.817 13.89 13.89 0 0 0-.53 3.606H3.047c.038-1.663.39-3.353 1.051-5.08a16.917 16.917 0 0 1 2.66-4.634H1.392zm9.363 2.245h12.99v11.255h-12.99zm.839 1.145v1.43h2.683l-3.023 5.472v1.534h4.675v-1.417H12.65l3.156-5.692V9.2zm5.034 0v8.436h1.164V9.195zm2.268 0v8.436h1.164V14.45h.76c.529 0 .93-.038 1.209-.12a1.5 1.5 0 0 0 .597-.398 2.298 2.298 0 0 0 .492-.831c.128-.356.192-.794.192-1.308a3.507 3.507 0 0 0-.332-1.64 1.637 1.637 0 0 0-.832-.832c-.211-.083-.673-.125-1.38-.125zm1.164 1.43h.567c.416 0 .695.018.831.056.19.049.348.174.469.374.125.2.189.454.189.756a1.516 1.516 0 0 1-.136.658.88.88 0 0 1-.363.415c-.151.087-.461.133-.918.133h-.639z"/></svg>
```
Comment (PeterShaggyNoble) [inline icons/7zip.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>7Zip</title><path d="M0 18.858h24V8.181H10.717V5.142H0ZM2.021 7.271h6.657v1.994c-1.74 2.09-2.84 4.502-2.948 7.404H3.477c.09-2.501.353-4.954 2.283-6.994l.033-.033H2.021Zm8.45 1.253h13.215v10.143H10.47Zm6.01 1.213v6.871h1.482v-6.87Zm2.755.043v6.912h1.616v-2.42h1.029c.43-.001.754-.29.969-.716.427-.848.429-2.257-.024-3.092-.227-.419-.571-.697-1.033-.684zm-7.924.002v1.596h2.217l-2.304 3.736v1.54h4.287V15.1h-2.698l2.786-3.909v-1.41Zm9.452 1.512h.595c.164-.006.287.081.371.217.17.273.172.736.004.99a.364.364 0 0 1-.373.176l-.55.047z"/></svg>
```

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]

**Review by adamrusted — APPROVED** [review]

**Pull request merged**